src/Maker/MakeTest.php

```diff
@@ -91,13 +93,13 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
         switch ($currentCommand) {
             case 'make:unit-test':
                 $input->setArgument('type', 'TestCase');
-                $io->caution('The "make:unit-test" command is deprecated, use "make:test" instead.');
+                $io->warning('The "make:unit-test" command is deprecated, use "make:test" instead.');
 
                 return;
 
             case 'make:functional-test':
                 $input->setArgument('type', trait_exists(PantherTestCaseTrait::class) ? 'WebTestCase' : 'PantherTestCase');
```

Comment:
Just caught this - shouldn't this be asked? What if I want a `WebTestCase` to check the status codes of a couple routes but want `PantherTestCase` for browser based tests.

Reply:
I guess even before this, if you used `make:functional-test`, we already looked to see if you had Panther available and then used it automatically. It's a minor thing, since this is deprecated. The new system of actually asking is much better. But I'll keep this the way it is now... which is the way it was before mostly.